src/sdl2/mixer/mod.rs

```diff
@@ -250,6 +250,27 @@ impl Chunk {
         }
     }
 
+    /// Load chunk from a static byte buffer.
+    pub fn from_static_bytes(buf: &'static [u8]) -> Result<Chunk, String> {
+        let rw = unsafe {
+            sys::SDL_RWFromConstMem(buf.as_ptr() as *const c_void, buf.len() as c_int)
+        };
+
+        if rw.is_null() {
+            return Err(get_error());
+        }
+
+        let raw = unsafe { mixer::Mix_LoadWAV_RW(rw, 0) };
+        if raw.is_null() {
+            Err(get_error())
+        } else {
+            Ok(Chunk {
+                raw: raw,
+                owned: true,
+            })
+        }
```

Comment:
~~I saw in different `from_*` methods that there was a private method to just do this stuff to avoid duplicates in the code. Any reason it isn't used there?~~

**EDIT**: I guess the code base is out-of-date just by looking at the extended diff… or at least different from the latest stable release. I guess this is also one of the things that should be checked after rebasing the fork and before merging it.